src/creosote/cli.py

```diff
@@ -1,19 +1,52 @@
 import argparse
-import glob
+import dataclasses
 import sys
+import typing
+from dataclasses import dataclass, field
 from enum import Enum
+from typing import List, Literal
 
+import toml
 from loguru import logger
 
 from creosote import formatters, parsers, resolvers
 from creosote.__about__ import __version__
 
 
+@dataclass
+class Config:
+    """Structured configuration data.
+
+    It is important that these attributes exactly match
+    the ``dest`` specified in ``add_argument``.
+    """
+
+    format: Literal["default", "no-color", "porcelain"] = "default"  # noqa: A003
+    paths: List[str] = field(default_factory=lambda: ["src"])
+    sections: List[str] = field(default_factory=lambda: ["project.dependencies"])
+    exclude_deps: List[str] = field(default_factory=list)
+    deps_file: str = "pyproject.toml"
+    venvs: List[str] = field(default_factory=lambda: [".venv"])
```

Comment:
I had requests in the past to auto-detect the currently activated virtual environment (#168). I was thinking the first value in this list could be `os.environ.get("VIRTUAL_ENV", ".venv")`.

```python
venvs: List[str] = field(default_factory=lambda: [os.environ.get("VIRTUAL_ENV", ".venv")])
```

Meaning; pick the path to the active virtual environment OR fall back to `".venv"`.

We can add this in as a separate change later, before the 3.0.0 release, but it should be part of the 3.0.0 release.

Do you foresee any issues with this? (I think it should be fine)
I suggest we leave a todo behind on this one:

```suggestion
    venvs: List[str] = field(default_factory=lambda: [".venv"])  # TODO: auto-detect active venv before releasing 3.0.0
```

Reply:
addressed in 4a4435a4639c2bdc17a97